check-format-cpp.sh

```
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Apache License 2.0
+
+project_directory="$(git rev-parse --show-toplevel)"
+
+pushd "${project_directory}" > /dev/null
+
+    clang-format -Werror --dry-run -style=file:thirdparty/clang/.clang-format $(find ~+ src/ include/ test/ bindings/python/src/ -name '*.cpp' -o -name '*.cxx' -o -name '*.hpp' -o -name '*.tpp')
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)